src/status_im/subs/root.cljs

```diff
@@ -151,8 +151,6 @@
 
 ;;wallet
 (reg-root-key-sub :wallet :wallet)
-(reg-root-key-sub :wallet/scanned-address :wallet/scanned-address)
-(reg-root-key-sub :wallet/create-account :wallet/create-account)
```

Comment:
:wallet/create-account appeared to be unused, 
other than that only removing :wallet/scanned-address here because I didn't want to risk breaking too much in one pr.